latest/index.bs

```diff
@@ -331,24 +317,917 @@ Conforming readers:
-It is a list of dictionaries. Each entry describes a single transformation and MUST contain the field "type".
-The value of "type" MUST be one of the elements of the `type` column in the table below.
-Additional fields for the entry depend on "type" and are defined by the column `fields`.
+"coordinateTransformations" describe the mapping between two coordinate systems (defined by "axes").
+For example, to map an array's discrete coordinate system to its corresponding physical coordinates.
+Coordinate transforms are in the "forward" direction. They represent functions from *points* in the
+input space to *points* in the output space. 
+
+
+- MUST contain the field "type".
+- MUST contain any other fields required by the given "type" (see table below).
+- MUST contain the field "output", unless part of a `sequence` or `inverseOf` (see details).
+- MUST contain the field "input", unless part of a `sequence` or `inverseOf` (see details).
+- MAY contain the field "name". Its value MUST be unique across all "name" fields for coordinate transformations.
+- Parameter values MUST be compatible with input and output space dimensionality (see details).
 
 <table>
-  <tr><th>`identity`    <td>                                                   <td>identity transformation, is the default transformation and is typically not explicitly defined
-  <tr><th>`translation` <td> one of: `"translation":List[float]`, `"path":str` <td>translation vector, stored either as a list of floats (`"translation"`) or as binary data at a location in this container (`path`). The length of vector defines number of dimensions. |
-  <tr><th>`scale`       <td> one of: `"scale":List[float]`, `"path":str`       <td>scale vector, stored either as a list of floats (`scale`) or as binary data at a location in this container (`path`). The length of vector defines number of dimensions. |
+  <tr><th>`identity`
+    <td> 
+    <td>The identity transformation is the default transformation and is typically not explicitly defined.
+  <tr><th>`mapIndex`
+    <td>`"mapIndex":List[number]`
+    <td> A `mapIndex` transformation specifies an axis permutation by reordering the input axes.
+  <tr><th>`mapAxis`
+    <td>`"mapAxis":Dict[String:String]`
+    <td> A `maxAxis` transformation specifies an axis permutation as a map between axis names.
+  <tr><th>`translation`
+    <td> one of: <br>`"translation":List[number]`, <br>`"path":str`
+    <td>translation vector, stored either as a list of numbers (`"translation"`) or as binary data at a location
+    in this container (`path`).
+  <tr><th>`scale`
+    <td> one of: <br>`"scale":List[number]`, <br>`"path":str`
+    <td>scale vector, stored either as a list of numbers (`scale`) or as binary data at a location in this
+    container (`path`).
+  <tr><th>`affine`
+    <td> one of: <br>`"affine":List[number]`, <br>`"path":str`
+    <td>affine transformation matrix stored as a flat array stored either with json uing the affine field
```

Comment:
yes. here's a good reason https://github.com/ome/ngff/issues/146

Reply:
If I understand #146 correctly, that is an argument for a nested json array (e.g. `[[1,2,3],[4,5,6]]`) instead of a separate Zarr array.

I think the separate Zarr array introduces quite an overhead (at least 2 requests) for a tiny piece of information.

Reply:
> that is an argument for a nested json array instead of a separate Zarr array.

correct, I must have misunderstood you.  For the question, "why store transform parameters as a zarr array", the reason is here: https://github.com/ome/ngff/issues/101#issuecomment-1100783401